gradle/scripts/lib/java.gradle

```diff
@@ -1,28 +1,36 @@
+rootProject.ext {
+    if (rootProject.hasProperty('buildJdkVersion')) {
+        def jdkVersion = rootProject.findProperty('buildJdkVersion')
+        if (buildJdkVersion != jdkVersion) {
+            buildJdkVersion = jdkVersion
+            logger.quiet("Overriding JDK for build with ${buildJdkVersion}")
+        } else {
+            buildJdkVersion = '15'
+        }
+    } else {
+        buildJdkVersion = '15'
+    }
+
+    if (rootProject.hasProperty('testJavaVersion')) {
+        def testVersion = rootProject.findProperty('testJavaVersion')
+        if (buildJdkVersion != testVersion) {
+            testJavaVersion = testVersion
+            logger.quiet("Overriding JRE for tests with ${testJavaVersion}")
+        } else {
+            testJavaVersion = buildJdkVersion
+        }
+    } else {
+        testJavaVersion = buildJdkVersion
```

Comment:
Is it better to use int value than string value?

Reply:
IIUC, we cannot define an integer global variable.
https://docs.gradle.org/current/dsl/org.gradle.api.plugins.ExtraPropertiesExtension.html

Reply:
Thanks for the link. TIL. 😄

Reply:
By the way, Integer seems working. `ExtraPropertiesExtension.set(java.lang.String, java.lang.Object)`

Reply:
Oh, it's working. Thanks! 😄